build.sbt

```diff
@@ -317,3 +286,47 @@ lazy val experimental = project
     osgiSettings
   )
   .dependsOn(coreJVM % "compile->compile;test->test")
+
+lazy val microsite = project
```

Comment:
You may also want to remove the entry from `plugins.sbt` file.

Reply:
Yeah good catch :+1: will do